sqld/src/replication/replica/injector/mod.rs

```diff
@@ -44,6 +45,24 @@ impl Injector {
         let buffer = FrameBuffer::default();
         let ctx = InjectorHookCtx::new(buffer.clone());
         std::fs::create_dir_all(path)?;
+
+        {
+            // create the replication table if it doesn't exist. We need to do that without hooks.
+            let connection = sqld_libsql_bindings::Connection::open(
+                path,
+                OpenFlags::SQLITE_OPEN_READ_WRITE
+                    | OpenFlags::SQLITE_OPEN_CREATE
+                    | OpenFlags::SQLITE_OPEN_URI
+                    | OpenFlags::SQLITE_OPEN_NO_MUTEX,
+                &TRANSPARENT_METHODS,
+                // safety: hook is dropped after connection
+                (),
+                DEFAULT_AUTO_CHECKPOINT,
```

Comment:
Autocheckpoint should be preventively disabled in this case. If we somehow end up having 999 pages in WAL while recreating this table (e.g. because the user mistakenly dropped it), this call will trigger an autocheckpoint outside of our virtual WAL. And that we don't want, especially if we later introduce checkpoint logic that only allows checkpointing frames that were backed up in S3.